lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -454,7 +502,7 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
       estimation
     )
 
-    media_event = serialize(msg)
+    media_event = notification_to_media_event(msg) |> MediaEvent.encode()
```

Comment:
I would favor `to_media_event` as we also have `to_rtc_track`.  There is also elixir `to_string`